public/locales/zh/translation.json

```diff
@@ -815,7 +815,7 @@
     "title": "设置",
     "general": "一般的",
     "actionItemCategories": "行动项目类别",
-    "updateOrganization": "更新组织",
+    "Edit Organization": "編輯組織",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.